addons/crm/models/crm_team.py

```diff
@@ -15,6 +15,7 @@ class Team(models.Model):
 
     use_leads = fields.Boolean('Leads', help="Check this box to filter and qualify incoming requests as leads before converting them into opportunities and assigning them to a salesperson.")
     use_opportunities = fields.Boolean('Pipeline', help="Check this box to manage a presales process with opportunities.")
+    alias_assigned_id = fields.Many2one('res.users', string="Assign to", store=True)
```

Comment:
I think we should use more standard odoo construct and use the alias_user_id field of alias model. As crm team inherits from mail.alias through the alias_id field you should be able to use directly alias_user_id field in crm.team model. Inherits adds the alias fields to the team model and everything should be managed directly. Maybe you could give it a try.